proposals/lip-0062.md

```diff
@@ -37,6 +43,10 @@ Introduction of another hash function to the Lisk protocol was suggested in [LIP
 
 [LIP 0008][lip-0008] removed the pre-hashing step from the Lisk signing protocol. This was mainly done as this step was deemed unnecessary and removing it could improve the theoretical security of the signing process. However this is not reflected by a practical improvement in protocol security as the SHA-256 hash function is considered collision resistant and secure. This hash function is used in several critical parts of many applications, including Lisk, and any future findings that SHA-256 is insecure would require changing the protocol throughout.
 
+### Signing Non-Protocol Messages
+
+When signing a message with the _sign message_ feature in Lisk Desktop, then the message is typically independet of any chain. Hence, no chain ID needs to be added to the message prior to hashing or signing. Therefore, we add an extra signing function for this use case that works the same as the regular Ed25519 signature scheme with pre-hashing but omits a chain ID. This function should be used in Lisk Desktop for the _sign message_ feature. The double pre-hashing scheme used in Lisk Commander should simply be removed without replacement as there is no use case for it.
```

Comment:
> ".....the regular Ed25519 signature scheme with pre-hashing...."
Just to clarify, here are we talking about single pre-hashing? 

Since we are using the terms "single pre-hashing" and "double pre-hashing", it would be good to write explicitly here also in this sentence for the sake of clarity maybe.

Reply:
good point. I added "single"